Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -22,8 +22,135 @@ private static void Test(string args, int expected)
 
         public static int FindDigit(string equation)
         {
-            // Add your code here.
-            throw new NotImplementedException();
+            // a * b = c
+            string a = "";
+            string b = "";
+            string c = "";
+            int n = equation.Length;
```

Comment:
add null check on equation. It will throw NullReferenceException while calculating the length of equation when it's value is null.